### PR TITLE
naomi 2.0 for 2021 estimates

### PR DESCRIPTION
**Pull request by jeffeaton**
This is a branch with updates to data inputs and outputs for the 2021 version of Naomi.  I can add more useful comments to this later.

Thanks,
Jeff

### CODE REVIEW

File: inst/metadata/colour_scales.csv [hunk 1]
```diff
@@ -1,5 +1,5 @@
 iso3,indicator,colour,min,max,invert_scale
-default,current_art,interpolateViridis,0,80000,TRUE
+default,art_current,interpolateViridis,0,80000,TRUE
 default,prevalence,interpolateMagma,0,0.5,TRUE
 default,art_coverage,interpolateViridis,0,1,FALSE
 default,vls,interpolateWarm,0,1,FALSE
```
Comment (r-ash) [inline inst/metadata/colour_scales.csv:5]: The references in this file to `vls` and `recent` need to be updated to `viral_suppression_plhiv` and `recent_infected`

File: inst/metadata/metadata.csv [hunk 2]
```diff
@@ -1,28 +1,28 @@
...
-output,barchart,plhiv,mean,lower,upper,indicator_id,3,t_(PLHIV),1,100,0
-output,barchart,incidence,mean,lower,upper,indicator_id,7,t_(INCIDENCE),1000,,0.00
-output,barchart,new_infections,mean,lower,upper,indicator_id,8,t_(NEW_INFECTIONS),100,100,0
-output,barchart,anc_prevalence,mean,lower,upper,indicator_id,9,t_(ANC_HIV_PREVALENCE),1,,0.00%
-output,barchart,anc_art_coverage,mean,lower,upper,indicator_id,10,t_(ANC_PRIOR_ART_COVERAGE),1,,0.0%
+data_type,plot_type,indicator,value_column,error_low_column,error_high_column,indicator_column,indicator_value,name,scale,accuracy,format
+survey,choropleth,prevalence,est,,,indicator,prevalence,t_(HIV_PREVALENCE),1,,0.00%
+survey,choropleth,art_coverage,est,,,indicator,art_coverage,t_(ART_COVERAGE),1,,0.00%
+programme,choropleth,art_current,art_current,,,,,t_(ART_NUMBER_ATTENDING),1,,0
+survey,choropleth,recent_infected,est,,,indicator,recent_infected,t_(PROPORTION_RECENTLY_INFECTED),100,100,0
```
Comment (r-ash) [inline inst/metadata/metadata.csv:5]: The `value_column` needs to be updated here too - in the test data this has been renamed to `estimate`.

**Review by r-ash — CHANGES_REQUESTED** [review]
Couple of comments from issues seen in the front end

File: inst/metadata/colour_scales.csv (hunk 1 above)
Comment (jeffeaton) [reply inst/metadata/colour_scales.csv]: Thanks very much for catching these. Addressed in 8a5c7f3.

File: inst/metadata/metadata.csv (hunk 2 above)
Comment (jeffeaton) [reply inst/metadata/metadata.csv]: Thanks very much for catching these. Addressed in 8a5c7f3.

**Review by r-ash — APPROVED** [review]

**Pull request merged**